app/views/components/_image_meta.html.erb

```diff
@@ -18,23 +22,25 @@
     </div>
     <div class="govuk-grid-column-two-thirds">
 
-      <%= render "components/metadata", {
-        inline: true,
-        items: [
-          {
-            field: t("documents.show.lead_image.fields.alt_text"),
-            value: alt_text
-          },
-          {
-            field: t("documents.show.lead_image.fields.caption"),
-            value: caption
-          },
-          {
-            field: t("documents.show.lead_image.fields.credit"),
-            value: credit
-          }
-        ]
-      } %>
+      <% if meta.any? %>
+        <%= render "components/metadata", {
+          inline: true,
+          items: [
+            {
+              field: t("documents.show.lead_image.fields.alt_text"),
+              value: meta[:alt_text]
+            },
+            {
+              field: t("documents.show.lead_image.fields.caption"),
+              value: meta[:caption]
+            },
+            {
+              field: t("documents.show.lead_image.fields.credit"),
+              value: meta[:credit]
+            }
```

Comment:
Are these ok to accept nil values?

Reply:
I think so, the metadata component supports nil for these and if we have a nil for `value` we still want to display the `field` label.